server/base/src/main/java/org/apache/accumulo/server/util/Admin.java

```diff
@@ -106,6 +106,15 @@ static class AdminOpts extends ServerUtilOpts {
     boolean force = false;
   }
 
+  @Parameters(commandDescription = "Compaction Temp Files Utility")
+  static class FindCompactionTmpFilesCommand {
```

Comment:
I wonder if you could just reuse FindCompactionTmpFiles.Opts. If you did this, then you could refactor FindCompactionTmpFiles.main and move most of the logic to a new method `FindCompactionTmpFiles.execute(Opts)`.